### PR TITLE
misc fixes

### PR DESCRIPTION
**Pull request by shixiedesign**
### Related Ticket(s)

Here are a list of small bugs discovered by Jonathan during component content model work. 

https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/issues/1178

### Changelog

**New**

- added a bit of guidance to `Content block horizontal`

**Changed**

- Fixed links to `video-player` to `video`

**Removed**

- Button group page
- Removed links to `Feature card block` for now

### CODE REVIEW

File: src/pages/components/leadspace.mdx [hunk 1]
```diff
@@ -73,32 +82,35 @@ This option has 4 empty columns on the left and right to accommodate pages that
...
+| CTA - Button                                                              | Component   | No       | Max of two buttons | XXS (40 / 55)                                  | –                                                      |
+| Gradient                                                                  | True/False  | No       | –                  | –                                              | True – Show gradient (default)<br/>False – No gradient |
+| Background [image](https://www.ibm.com/standards/carbon/components/image) | Component   | No       | 1                  | –                                              | Breakpoints require different images.                  |
+| Type                                                                      | Option list | Yes      | –                  | –                                              | Center, Left, or Small                                 |
 
-More more on character count, <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer">see all character count standards</a>.
+More more on character count,
```
Comment (janchild) [inline src/pages/components/leadspace.mdx:104]:
```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).
```

File: src/pages/components/leadspace.mdx [hunk 2]
```diff
@@ -73,32 +82,35 @@ This option has 4 empty columns on the left and right to accommodate pages that
...
+<a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer">see
```
Comment (janchild) [inline src/pages/components/leadspace.mdx:105]:
```suggestion
```

File: src/pages/components/leadspace.mdx [hunk 3]
```diff
@@ -73,32 +82,35 @@ This option has 4 empty columns on the left and right to accommodate pages that
...
+all character count standards</a>.
```
Comment (janchild) [inline src/pages/components/leadspace.mdx:106]:
```suggestion
```

File: src/pages/components/leadspace.mdx [hunk 4]
```diff
@@ -108,7 +120,8 @@ There are two themes to choose from when using the Lead space:
 
 ### Dark themes (gray 100 or gray 90)
```
Comment (janchild) [inline src/pages/components/leadspace.mdx:121]:
```suggestion
### Dark themes (Gray 100 or Gray 90)
```

File: src/pages/components/leadspace.mdx [hunk 5]
```diff
@@ -108,7 +120,8 @@ There are two themes to choose from when using the Lead space:
...
 
-Use `gray 100` or `gray 90` when the background image has shadows, is dimly lit, or uses a dark color palette.
+Use `gray 100` or `gray 90` when the background image has shadows, is dimly lit,
```
Comment (janchild) [inline src/pages/components/leadspace.mdx:123]:
```suggestion
Use `Gray 100` or `Gray 90` when the background image has shadows, is dimly lit,
```

File: src/pages/components/leadspace.mdx [hunk 6]
```diff
@@ -119,7 +132,8 @@ Use `gray 100` or `gray 90` when the background image has shadows, is dimly lit,
 
 ### Light themes (white or gray 10)
```
Comment (janchild) [inline src/pages/components/leadspace.mdx:133]:
```suggestion
### Light themes (White or Gray 10)
```

File: src/pages/components/leadspace.mdx [hunk 7]
```diff
@@ -119,7 +132,8 @@ Use `gray 100` or `gray 90` when the background image has shadows, is dimly lit,
...
 
-Use `white` or `gray 10` when the background image is well-lit or brightly colored.
+Use `white` or `gray 10` when the background image is well-lit or brightly
```
Comment (janchild) [inline src/pages/components/leadspace.mdx:135]:
```suggestion
Use `White` or `Gray 10` when the background image is well-lit or brightly
```

File: src/pages/components/video.mdx [hunk 8]
```diff
@@ -1,11 +1,13 @@
 ---
 title: Video
-description: The Video component allows you as an adopter to embed Kaltura videos from the media center in various ways throughout a page.
+description:
+  The Video component allows you as an adopter to embed Kaltura videos from the
```
Comment (janchild) [inline src/pages/components/video.mdx:4]:
```suggestion
  The Video component allows you to embed Kaltura videos from the
```

File: src/pages/components/video.mdx [hunk 9]
```diff
@@ -22,26 +24,32 @@ import ResourceLinks from "components/ResourceLinks";
...
-The video component defaults to the aspect ratio of the source video, or one of the standard aspect ratios specified below can be used instead. In most cases 16x9 is recommended.
+The video component defaults to the aspect ratio of the source video, or one of
+the standard aspect ratios specified below can be used instead. In most cases
+16x9 is recommended.
 
-By default, the component is prefilled with content returned from the Kaltura service. You can override this default content and provide a custom video title, description, and poster image. If the content is optional for the component you're using, you could also remove the prefilled content.
+By default, the component is prefilled with content returned from the Kaltura
+service. You can override this default content and provide a custom video title,
+description, and poster image. If the content is optional for the component
+you're using, you could also remove the prefilled content.
```
Comment (janchild) [inline src/pages/components/video.mdx:34]:
```suggestion
you're using, you can also remove the prefilled content.
```

File: src/pages/components/callout-with-media.mdx [hunk 10]
```diff
@@ -22,20 +24,23 @@ import ResourceLinks from "components/ResourceLinks";
...
+| Headline                                                                                                                                                   | Short copy | Yes      | 1           | XXS (40 / 55)                             |
+| Paragraph                                                                                                                                                  | Long copy  | No       | xe 1        | XXL (350 / 450)                           |
+| [Video](https://www.ibm.com/standards/carbon/components/video) or [Image with caption](https://www.ibm.com/standards/carbon/components/image-with-caption) | Component  | Yes      | 1           | –                                         |
 
-To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
+To see the list of all character count standards, go to this
```
Comment (janchild) [inline src/pages/components/callout-with-media.mdx:43]:
```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

```

File: src/pages/components/callout-with-media.mdx [hunk 11]
```diff
@@ -22,20 +24,23 @@ import ResourceLinks from "components/ResourceLinks";
...
+<a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
```
Comment (janchild) [inline src/pages/components/callout-with-media.mdx:44]:
```suggestion
```

File: src/pages/components/content-block-horizontal.mdx [hunk 12]
```diff
@@ -30,15 +40,15 @@ import ResourceLinks from "components/ResourceLinks";
...
+| ---------------------------------------------------------------------------------------------------- | ---------- | -------- | ----------- | ----------------------------------------- |
+| Headline                                                                                             | Short copy | Yes      | 1           | XXS (40 / 55)                             |
+| TOC headline override                                                                                | Short copy | No       | 1           | Mini (25 / 35)                            |
 | [Content item – horizontal](https://www.ibm.com/standards/carbon/components/content-item-horizontal) | Component  | Yes      | 1-8         | –                                         |
 
-To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
+To see the list of all character count standards, go to this
```
Comment (janchild) [inline src/pages/components/content-block-horizontal.mdx:51]:
```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

```

File: src/pages/components/content-block-horizontal.mdx [hunk 13]
```diff
@@ -30,15 +40,15 @@ import ResourceLinks from "components/ResourceLinks";
...
+<a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
```
Comment (janchild) [inline src/pages/components/content-block-horizontal.mdx:52]:
```suggestion
```

File: src/pages/components/content-block-media.mdx [hunk 14]
```diff
@@ -33,22 +37,24 @@ By default the Content block media starts with a brief overview before going int
...
+| Headline                                                                                       | Short copy | Yes      | 1           | XXS (40 / 55)                             |
+| Paragraph                                                                                      | Long copy  | No       | 1           | XL (250 / 325)                            |
+| [Content group – simple](https://www.ibm.com/standards/carbon/components/content-group-simple) | Component  | Yes      | Less than 4 | –                                         |
+| Featured card – medium                                                                         | Component  | No       | 1           | –                                         |
+| [Link list](https://www.ibm.com/standards/carbon/components/link-list)                         | Component  | No       | 1           | –                                         |
 
-You can see the list of all character count standards <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer">here</a>.
+You can see the list of all character count standards
```
Comment (janchild) [inline src/pages/components/content-block-media.mdx:57]:
```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

```

File: src/pages/components/content-block-media.mdx [hunk 15]
```diff
@@ -33,22 +37,24 @@ By default the Content block media starts with a brief overview before going int
...
+<a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer">here</a>.
```
Comment (janchild) [inline src/pages/components/content-block-media.mdx:58]:
```suggestion
```

File: src/pages/components/content-block-segmented.mdx [hunk 16]
```diff
@@ -33,23 +38,26 @@ Default Content block segmented allows you to break up the narrative content int
...
+| [Image with caption](https://www.ibm.com/standards/carbon/components/image-with-caption) or [Video](https://www.ibm.com/standards/carbon/components/video) | Component  | No       | 1           | –                                              |
+| [Content group – simple](https://www.ibm.com/standards/carbon/components/content-group-simple)                                                             | Component  | Yes      | 2–3         | –                                              |
+| [CTA (card)](https://www.ibm.com/standards/carbon/components/cta/#card)                                                                                    | Component  | No       | 1           | –                                              |
+| [Link list](https://www.ibm.com/standards/carbon/components/link-list)                                                                                     | Component  | No       | 1           | –                                              |
+
+To see the list of all character count standards, go to this
```
Comment (janchild) [inline src/pages/components/content-block-segmented.mdx:60]:
```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

```

File: src/pages/components/content-block-segmented.mdx [hunk 17]
```diff
@@ -33,23 +38,26 @@ Default Content block segmented allows you to break up the narrative content int
...
+<a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer">link</a>.
```
Comment (janchild) [inline src/pages/components/content-block-segmented.mdx:61]:
```suggestion
```

File: src/pages/components/content-block-simple.mdx [hunk 18]
```diff
@@ -33,23 +37,27 @@ Content block simple is a top-level pattern meaning it ideally lives at the top
...
+| Paragraph                                                                                                                                                  | Long copy  | Yes      | 1           | XXXXL (1600 / 2400)                       |
+| [Image with caption](https://www.ibm.com/standards/carbon/components/image-with-caption) or [Video](https://www.ibm.com/standards/carbon/components/video) | Component  | No       | 1           | –                                         |
+| [CTA (card)](https://www.ibm.com/standards/carbon/components/cta)                                                                                          | Component  | No       | 1           | –                                         |
+| [Link list](https://www.ibm.com/standards/carbon/components/link-list)                                                                                     | Component  | No       | 1           | –                                         |
+
+To see the list of all character count standards, go to this
```
Comment (janchild) [inline src/pages/components/content-block-simple.mdx:60]:
```suggestion
```

File: src/pages/components/content-block-simple.mdx [hunk 19]
```diff
@@ -33,23 +37,27 @@ Content block simple is a top-level pattern meaning it ideally lives at the top
...
+<a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
```
Comment (janchild) [inline src/pages/components/content-block-simple.mdx:61]:
```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

```

File: src/pages/components/content-group-simple.mdx [hunk 20]
```diff
@@ -22,25 +24,28 @@ import ResourceLinks from "components/ResourceLinks";
...
+|                                                                                                                                                            | Paragraph                                                               | Long copy  | Yes      | 1           | XXXXL (1600 / 2400)                            |
+|                                                                                                                                                            | [CTA (text)](https://www.ibm.com/standards/carbon/components/cta/#text) | Component  | No       | 1           | –                                              |
+| [CTA (card)](https://www.ibm.com/standards/carbon/components/cta/#card)                                                                                    |                                                                         | Component  | No       | 1           | –                                              |
+
+To see the list of all character count standards, go to this
```
Comment (janchild) [inline src/pages/components/content-group-simple.mdx:48]:
```suggestion
```

File: src/pages/components/content-group-simple.mdx [hunk 21]
```diff
@@ -22,25 +24,28 @@ import ResourceLinks from "components/ResourceLinks";
...
+<a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer">link</a>.
```
Comment (janchild) [inline src/pages/components/content-group-simple.mdx:49]:
```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

```

File: src/pages/components/cta-section.mdx [hunk 22]
```diff
@@ -22,25 +26,30 @@ import ResourceLinks from "components/ResourceLinks";
...
+|                                                                                                         | [CTA (text)](https://www.ibm.com/standards/carbon/components/cta/#text) | Component  | Yes      | 1                 | –                                              | –                                                              |
+| [Link list (end of section)](https://www.ibm.com/standards/carbon/components/link-list/#end-of-section) |                                                                         | Component  | No       | 1                 | –                                              | You can choose either Content items or the Link list component |
+
+To see the list of all character count standards, go to this
+<a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer">link</a>.
```
Comment (janchild) [inline src/pages/components/cta-section.mdx:53]:
```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

```

File: src/pages/components/cta-section.mdx [hunk 23]
```diff
@@ -22,25 +26,30 @@ import ResourceLinks from "components/ResourceLinks";
...
+|                                                                                                         | Paragraph                                                               | Long copy  | Yes      | 1                 | L (150 / 200)                                  | –                                                              |
+|                                                                                                         | [CTA (text)](https://www.ibm.com/standards/carbon/components/cta/#text) | Component  | Yes      | 1                 | –                                              | –                                                              |
+| [Link list (end of section)](https://www.ibm.com/standards/carbon/components/link-list/#end-of-section) |                                                                         | Component  | No       | 1                 | –                                              | You can choose either Content items or the Link list component |
+
+To see the list of all character count standards, go to this
```
Comment (janchild) [inline src/pages/components/cta-section.mdx:52]:
```suggestion
```

File: src/pages/components/lead-space-block.mdx [hunk 24]
```diff
@@ -22,26 +24,32 @@ import ResourceLinks from "components/ResourceLinks";
...
+| Paragraph                                                                                                                                                  | Long copy  | Yes      | 1           | XL (250 / 325)                            |
+| [Video](https://www.ibm.com/standards/carbon/components/video) or [Image with caption](https://www.ibm.com/standards/carbon/components/image-with-caption) | Media      | No       | 1           | –                                         |
+| [Link list (End of section)](https://www.ibm.com/standards/carbon/components/link-list/)                                                                   | Component  | No       | 1           | –                                         |
+| [CTA (button)](https://www.ibm.com/standards/carbon/components/cta/)                                                                                       | Component  | No       | 1           | –                                         |
+
+To see the list of all character count standards, go to this
+<a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank">link</a>.
```
Comment (janchild) [inline src/pages/components/lead-space-block.mdx:53]:
```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

```

File: src/pages/components/lead-space-block.mdx [hunk 25]
```diff
@@ -22,26 +24,32 @@ import ResourceLinks from "components/ResourceLinks";
...
+| Paragraph                                                                                                                                                  | Long copy  | Yes      | 1           | XL (250 / 325)                            |
+| [Video](https://www.ibm.com/standards/carbon/components/video) or [Image with caption](https://www.ibm.com/standards/carbon/components/image-with-caption) | Media      | No       | 1           | –                                         |
+| [Link list (End of section)](https://www.ibm.com/standards/carbon/components/link-list/)                                                                   | Component  | No       | 1           | –                                         |
+| [CTA (button)](https://www.ibm.com/standards/carbon/components/cta/)                                                                                       | Component  | No       | 1           | –                                         |
+
+To see the list of all character count standards, go to this
```
Comment (janchild) [inline src/pages/components/lead-space-block.mdx:52]:
```suggestion
```

File: src/pages/components/leaving-ibm.mdx [hunk 26]
```diff
@@ -1,11 +1,12 @@
 ---
-title: Leaving IBM
-description: The Leaving IBM displays as an overlay when users click an external link icon.
+title: Leaving IBM modal
+description:
+  The Leaving IBM displays as an overlay when users click an external link icon.
```
Comment (janchild) [inline src/pages/components/leaving-ibm.mdx:4]:
```suggestion
  The Leaving IBM notice displays as an overlay when users click an external link icon.
```

File: src/pages/components/video.mdx [hunk 27]
```diff
@@ -52,6 +60,7 @@ The video can be played in place of the poster image, or inside the [Lightbox Me
 | Poster image | Image       | No       | 1           | Default to poster image from video source                       |
 | Caption      | Short copy  | No       | 1           | Default to existing caption from video source                   |
 
-To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
+To see the list of all character count standards, go to this
+<a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
```
Comment (janchild) [inline src/pages/components/video.mdx:64]:
```suggestion
For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).

```

File: src/pages/components/video.mdx [hunk 28]
```diff
@@ -52,6 +60,7 @@ The video can be played in place of the poster image, or inside the [Lightbox Me
 | Poster image | Image       | No       | 1           | Default to poster image from video source                       |
 | Caption      | Short copy  | No       | 1           | Default to existing caption from video source                   |
 
-To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
+To see the list of all character count standards, go to this
```
Comment (janchild) [inline src/pages/components/video.mdx:63]:
```suggestion
```

**Review by janchild — COMMENTED** [review]
Lots of nice fixes on lots of pages here! I did a super detailed review and found some extra things too. Sorry should have used the `Edit file` option.

File: src/pages/components/content-block-horizontal.mdx [hunk 29]
```diff
@@ -26,7 +26,7 @@ import ResourceLinks from 'components/ResourceLinks';
 
 On a page with largely vertical content, Content block horizontal presents a
 moment to enhance vertical ryhthm by switch up the reading direction. It is also
-a component good for presenting large amount of CTAs with context. Please note
+a good component for presenting large amount of CTAs with context. Please note
```
Comment (JBrandwine) [inline src/pages/components/content-block-horizontal.mdx:29]: a good component for presenting large amount of CTAs with context.

**Review by JBrandwine — CHANGES_REQUESTED** [review]
a good component for presenting a large amount of CTAs with context.

File: src/data/nav-items.yaml [hunk 30]
```diff
@@ -120,7 +118,7 @@
     - title: Lead space search
       path: /components/lead-space-search
     - title: Leaving IBM
```
Comment (janchild) [inline src/data/nav-items.yaml:120]:
```suggestion
    - title: Leaving IBM modal
```

File: src/data/components.json [hunk 31]
```diff
@@ -435,7 +408,7 @@
       }
     },
     "Leaving IBM": {
```
Comment (janchild) [inline src/data/components.json:410]:
```suggestion
    "Leaving IBM modal": {
```

**Review by janchild — COMMENTED** [review]
@shixiedesign I noticed that the .yaml file wasn't updated in the preview. Not sure if it's a caching issue. But I noticed the component name needed an update, so just two more changes here.

**Review by JBrandwine — APPROVED** [review]
Content block horizontal text looks good, updated link to video component validated on callout with media, button group page not present in left nav.

Thanks!

**Review by janchild — APPROVED** [review]
🧗‍♀️ 🥂

**Pull request merged**